docs/src/dataloader.md

```diff
@@ -52,7 +51,28 @@ var result = executer.ExecuteAsync(opts => {
 });
 ```
 
-## Usage
+## Example of Setup *without* Dependency injection
+
+1. Instantiate a `DataLoaderContextAccessor` and a `DataLoaderContext` and `DataLoaderDocumentListener`
+in the GraphQL controller. Perhaps like this:
+``` chsarp
+    var ctxAc = new DataLoader.DataLoaderContextAccessor();
```

Comment:
In general I would also prefer the docs to use descriptive names vs non descriptive.  `ctxAc` is non descriptive.